src/linux/wifi/core/AccessPointControllerLinux.cxx

```diff
@@ -119,6 +166,20 @@ AccessPointControllerLinux::GetIsEnabled()
     return isEnabled;
 }
 
+bool
+AccessPointControllerLinux::SetPhyType(Dot11PhyType phyType)
+{
+    Wpa::HostapdHwMode hwMode = detail::Dot11PhyTypeToHostapdHwMode(phyType);
+    const auto& propertyName = Wpa::ProtocolHostapd::PropertyNameSetHwMode;
+    const auto& propertyValue = detail::HostapdHwModeToPropertyValue(hwMode);
+
+    try {
+        return m_hostapd.SetProperty(propertyName, propertyValue);
+    } catch (const Wpa::HostapdException& ex) {
+        throw AccessPointControllerException(std::format("Failed to set PHY type for interface {} ({})", GetInterfaceName(), ex.what()));
+    }
+}
```

Comment:
Thanks, I'll need to add onto this a bit more because some PHY types need to set more than just `hw_mode`, such as `ieee80211n=1` or `ieee80211ac=1` for n/ac PHY types